metis_client/dtos/datasource.py

```diff
@@ -18,16 +18,16 @@
 
 
 class DataSourceType(int, Enum):
-    "Data source types"
+
```

Comment:
Why docstring remove?

Reply:
because it doesn’t bring any valuable information here

Reply:
Correct. So there are two ways to go about it: write valuable docstring or turn off the linter rule [missing-class-docstring](https://pylint.pycqa.org/en/latest/user_guide/messages/convention/missing-class-docstring.html).